test/spectral/tests/custom-rules/ibm-content-type-is-specific.test.js

```diff
@@ -0,0 +1,73 @@
+const inCodeValidator = require('../../../../src/lib');
+
+describe('spectral - test content type is not */*', function() {
+  it('should error only when the content type is */*', async () => {
+    const spec = {
+      openapi: '3.0.0',
+      paths: {
+        path1: {
+          post: {
+            requestBody: {
+              content: {
+                // error 1
+                '*/*': {
+                  $ref: '#/components/schemas/GenericSchema'
+                }
+              }
+            },
+            parameters: [
+              {
+                name: 'param1',
+                in: 'query',
+                content: {
+                  // error 2
+                  '*/*': {
+                    $ref: '#/components/schemas/GenericSchema'
+                  }
+                }
+              }
+            ],
+            responses: {
+              '200': {
+                content: {
+                  // error 3
+                  '*/*': {
+                    $ref: '#/components/schemas/GenericSchema'
+                  }
+                }
+              },
+              '201': {
+                content: {
+                  // no error
+                  'application/json': {
+                    $ref: '#/components/schemas/GenericSchema'
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      components: {
```

Comment:
Please a "no error" case for a content type like `image/*` or `text/*` -- just to make clear that these are acceptable.